v0.7/language/bert/accuracy-squad.py

```diff
@@ -336,8 +346,11 @@ def main():
     parser.add_argument("--out_file", default="build/result/predictions.json", help="Path to output predictions file")
     parser.add_argument("--features_cache_file", default="eval_features.pickle", help="Path to features' cache file")
     parser.add_argument("--output_transposed", action="store_true", help="Transpose the output")
+    parser.add_argument("--output_dtype", default="float16", choices=dtype_map.keys(), help="Output data type")
```

Comment:
@nvzhihanj What was the original default? If it was `np.float32`, we should keep the default unchanged

Reply:
Changed.